src/themes/ec/variables/_tag.scss

```diff
@@ -2,6 +2,7 @@
 @use '../index' as *;
 
 $tag: (
+  font: var(--f-ui-s, var(--f-s)),
```

Comment:
mmh..this doesn't make much sense, this was defined in the component scss so that if --f-ui-s was defined it would have been used otherwise the fallback" would have been used instead.
It was a not super nice way to set two different value in the two systems, because EC doesn't have the ui font, so it would have used the other.
If you move this in the system variables, then there is no need to use a fallback, in this case in EC is --f-s and in EU is --f-ui-s